src/plugins/mf_priority.cpp

```diff
@@ -25,8 +25,11 @@ extern "C" {
 #include <cassert>
 #include <algorithm>
 #include <cinttypes>
+#include <vector>
 
 std::map<int, std::map<std::string, double>> users;
+std::map<int, std::map<std::string, std::vector<int>>> users_mj;
```

Comment:
Suggestion: consider renaming `users_mj` to something more descriptive, since it is a global variable. Perhaps just `user_max_jobs`. Add a comment describing the global variable (e.g. "Mapping of user,bank to max jobs for that user,bank pair")

(You could also separately add a comment for the other globals here)